From 999a97bc2403987aebd500996532d93404dd024b Mon Sep 17 00:00:00 2001
From: rleix <rayx.lei@intel.com>
Date: Fri, 20 Jul 2018 13:30:03 +0800
Subject: [PATCH] Fix the display issue for AlertDialog's button

Add the padding between Button3(NeutralButton) and Button2(NegativeButton),
avoiding these two buttons display so closely.

Change-Id: I20431208abf85acef4862d8dcf752447b812b34b
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-65784
Signed-off-by: Lei,RayX <rayx.lei@intel.com>
---
 .../base/core/res/res/layout/alert_dialog_button_bar_material.xml       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/car_product/overlay/frameworks/base/core/res/res/layout/alert_dialog_button_bar_material.xml b/car_product/overlay/frameworks/base/core/res/res/layout/alert_dialog_button_bar_material.xml
index 7203849..3bb04c0 100644
--- a/car_product/overlay/frameworks/base/core/res/res/layout/alert_dialog_button_bar_material.xml
+++ b/car_product/overlay/frameworks/base/core/res/res/layout/alert_dialog_button_bar_material.xml
@@ -37,6 +37,7 @@
             android:id="@+id/button3"
             style="@style/CarAction1"
             android:background="@drawable/dialog_button_background"
+            android:layout_marginEnd="@dimen/car_padding_4"
             android:layout_width="wrap_content"
             android:layout_height="@dimen/car_card_action_bar_height" />
 
@@ -52,7 +53,6 @@
             android:id="@+id/button1"
             style="@style/CarAction1"
             android:background="@drawable/dialog_button_background"
-            android:layout_marginStart="@dimen/car_padding_4"
             android:layout_width="wrap_content"
             android:layout_height="@dimen/car_card_action_bar_height" />
         <Space
-- 
1.9.1

